source/features/repo-age.tsx

```diff
@@ -15,15 +15,21 @@ const dateFormatter = new Intl.DateTimeFormat('en-US', {
 });
 
 const getRepoCreationDate = cache.function(async (): Promise<string> => {
-	const {repository} = await api.v4(`
-		repository(${getRepoGQL()}) {
-			createdAt
-		}
-	`);
+	const commitsCount = Number(select('li.commits .num')!.textContent!.trim()!.replace(',', ''));
+	const lastCommitHash = select('.commit-tease .commit-tease-sha')!.getAttribute('href')!.split('/').pop();
```

Comment:
Shorter selector; use props instead of attributes where available

```suggestion
	const lastCommitHash = select('.commit-tease-sha')!.href.split('/').pop();
```